superset-frontend/src/views/components/Menu.tsx

```diff
@@ -97,6 +96,17 @@ const StyledHeader = styled.header`
     display: flex;
     flex-direction: column;
     justify-content: center;
+    min-height: ${({ theme }) => `${theme.gridUnit * 12.5}px`};
+    padding: ${({ theme }) =>
+      `${theme.gridUnit}px ${theme.gridUnit * 2}px ${theme.gridUnit}px ${
+        theme.gridUnit * 4
+      }px`};
+    max-width: ${({ theme }) => `${theme.gridUnit * 37}px`};
```

Comment:
The problem with using the `APP_ICON_WIDTH` here is that it would not behave the same as it used to. I think best is not to use it or change it with `APP_ICON_MAX_WIDTH` for clarity, if that can be of any benefit to the users